### PR TITLE
Add reset all statistics button on the curl manager report. Log the r…

### PR DESCRIPTION
**Pull request by xuangui2018**
…eset event when resetting. #17

### CODE REVIEW

File: classes/event/curlmanager_stats_reset.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        $this->data['edulevel'] = self::LEVEL_OTHER;
+    }
+
+    /**
+     * Returns description of what happened.
+     *
+     * @return string
+     */
+    public function get_description() : string {
+        return get_string('event:logresetstatseventdescription', 'tool_curlmanager');
```
Comment (brendanheywood) [inline classes/event/curlmanager_stats_reset.php]: This is one of the rare times when you should not use a lang string:

https://github.com/moodle/moodle/blob/master/lib/classes/event/base.php#L339-L343

File: report.php [hunk 2]
```diff
@@ -56,6 +67,22 @@
...
+$urlresetallcspstatistics = new moodle_url($PAGE->url, array(
+    'resetallcurlstatistics' => 1,
+    'sesskey' => sesskey(),
+));
+echo $OUTPUT->single_button($urlresetallcspstatistics,
+                get_string('resetallcurlstatistics', 'tool_curlmanager'),
+                'post',
+                [
+                    'actions' => [$action],
+                    'style' => 'float: right;'
```
Comment (brendanheywood) [inline report.php]: I don't think this works (at least in moodle), and either way please remove the float

Comment (xuangui2018) [reply report.php]: No this didn't work. I have removed float right.

File: classes/event/curlmanager_stats_reset.php (hunk 1 above)
Comment (xuangui2018) [reply classes/event/curlmanager_stats_reset.php]: Please check again.

**Pull request merged**